runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -1687,6 +1687,76 @@ J9::SymbolReferenceTable::findOrCreateGlobalFragmentSymbolRef()
    }
 
 
+TR::SymbolReference *
+J9::SymbolReferenceTable::createKnownStaticReferenceSymbolRef(void *dataAddress, TR::KnownObjectTable::Index knownObjectIndex)
+   {
+   char *name = "<known-static-reference>";
+   if (knownObjectIndex != TR::KnownObjectTable::UNKNOWN)
+      {
+      name = (char*)trMemory()->allocateMemory(25, heapAlloc);
+      sprintf(name, "<known-obj%d>", knownObjectIndex);
+      }
+   TR::StaticSymbol * sym = TR::StaticSymbol::createNamed(trHeapMemory(), TR::Address, dataAddress,name);
+   return TR::SymbolReference::create(self(), sym, knownObjectIndex);
+   }
+
+
+TR::SymbolReference *
+J9::SymbolReferenceTable::findOrCreateCurrentTimeMaxPrecisionSymbol()
+   {
+   if (!element(currentTimeMaxPrecisionSymbol))
+      {
+      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
+      sym->setHelper();
+
+      element(currentTimeMaxPrecisionSymbol) = new (trHeapMemory()) TR::SymbolReference(self(), currentTimeMaxPrecisionSymbol, sym);
+      }
+   return element(currentTimeMaxPrecisionSymbol);
+   }
+
+
+TR::SymbolReference *
+J9::SymbolReferenceTable::findOrCreateSinglePrecisionSQRTSymbol()
```

Comment:
I think this one was supposed to stay in OMR too...

Reply:
Agreed. Pushing changes soon.